devon/conf/devon.properties

```diff
@@ -15,3 +15,5 @@
 
 # In case you are sitting behind a proxy these JVM options may help:
 #export JAVA_OPTS=-Dhttp.proxyHost=myproxy.com -Dhttp.proxyPort=8080
+
+DEVON_IDE_TOOLS=(java mvn eclipse)
```

Comment:
this one is unintendet and needs to be reverted.

Reply:
seems vice versa. The changes I wanted to have on this PR are already on master and this change that I did on master is on the feature-branch. Very strange....